fdbserver/DataDistribution.actor.cpp

```diff
@@ -656,7 +666,7 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 	    optimalTeamCount(0), recruitingStream(0), restartRecruiting(SERVER_KNOBS->DEBOUNCE_RECRUITING_DELAY),
 	    unhealthyServers(0), includedDCs(includedDCs), otherTrackedDCs(otherTrackedDCs),
 	    zeroHealthyTeams(zeroHealthyTeams), zeroOptimalTeams(true), primary(primary),
-	    processingUnhealthy(processingUnhealthy) {
+		processingUnhealthy(processingUnhealthy) {
```

Comment:
Code format got messed up here. You can use pre-commit hook to clang-format lines you touched: https://github.com/barisione/clang-format-hooks